robber/bad_expectation.py

```diff
@@ -4,6 +4,7 @@ class BadExpectation(Exception):
     """
 
     def __init__(self, message):
+        super(BadExpectation, self).__init__(message)
```

Comment:
Actually, calling `super()` when subclassing is a _de facto_ practice. It's just that it tends to slip through your eyes when reviewing the code...